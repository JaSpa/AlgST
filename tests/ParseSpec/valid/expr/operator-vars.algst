-- Check that in an expression context parenthesized operators are parsed as
-- variable names.
(+) (*) (-) (/)
