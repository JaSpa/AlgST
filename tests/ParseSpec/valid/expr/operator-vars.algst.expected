Exp.App
|
+- Exp.App
|  |
|  +- Exp.App
|  |  |
|  |  +- Exp.Var (+)
|  |  |
|  |  `- Exp.Var (*)
|  |
|  `- Exp.Var (-)
|
`- Exp.Var (/)

